src/ex1/style.css

```diff
@@ -6,6 +6,15 @@
     background-color: aliceblue;
     width: 500px;
 }
+.pendingTasksCounter-visible{
```

Comment:
naming convention - you could just name this class visible/hidden because it may be used in the future by another logic
and also omit the text-align: center; part :)